src/plugins/workspace/server/saved_objects/workspace_id_consumer_wrapper.ts

```diff
@@ -37,6 +41,24 @@ export class WorkspaceIdConsumerWrapper {
       ...(finalWorkspaces.length ? { workspaces: finalWorkspaces } : {}),
     };
   }
+  private isDataSourceType(type: SavedObjectsFindOptions['type']): boolean {
```

Comment:
The type could also be the object type of SavedObjectsBulkCreateObject ? In this case, type will only be single string.

Reply:
The `isDataSourceType` is used within formatFindParams, whose `type` may be string[].